app/models/wallet_target.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class WalletTarget < ApplicationRecord
+  include PaperTrailTraceable
+
+  belongs_to :wallet
+  belongs_to :billable_metric, optional: true
```

Comment:
I'm a little surprised `billable_metric` can be optional, I don't understand what the entry would mean 🤔 

By the way, in the migration the billable_metric is `null: false`.

EDIT: Is it because there are more relationship possibly coming? In this case, why not use polymorphism?